### PR TITLE
Bug 533393 - Legal Documentation

### PR DESCRIPTION
**Pull request by squarti**

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=533393

File: bundles/org.eclipse.orion.client.ui/web/mixloginstatic/javascript/register.js [hunk 1]
```diff
@@ -11,8 +11,19 @@
 /*eslint-env amd, browser*/
 /*global URL*/
 define(['domReady', 'orion/xhr', 'orion/xsrfUtils', './common'], function(domReady, xhr, xsrfUtils, common) {
+	
+	function confirmAggrement() {
```
Comment (mrennie) [inline bundles/org.eclipse.orion.client.ui/web/mixloginstatic/javascript/register.js]: Did you mean to name this 'confirmAgreement'?

File: bundles/org.eclipse.orion.client.ui/web/mixloginstatic/javascript/register.js [hunk 2]
```diff
@@ -11,8 +11,19 @@
 /*eslint-env amd, browser*/
 /*global URL*/
 define(['domReady', 'orion/xhr', 'orion/xsrfUtils', './common'], function(domReady, xhr, xsrfUtils, common) {
+	
+	function confirmAgreement() {
+		if (!document.getElementById("aggrement").checked) {
```
Comment (carmacleod) [inline bundles/org.eclipse.orion.client.ui/web/mixloginstatic/javascript/register.js]: aggrement => agreement

File: bundles/org.eclipse.orion.client.ui/web/mixloginstatic/register.html [hunk 3]
```diff
@@ -59,8 +59,14 @@ <h1 class="login-title" id="emailLogin">Registration</h1>
 
 									<div class="form-elements-container email-container" id="emailContainer">
 										<input class="login-field" type="email" id="email" placeholder="Email" />
+										<div class="email-warning">The Eclipse community is an open and transparent community. Most of what you submit on our site will be visible to everyone, and your email address may be visible to users who use Bugzilla, Gerrit, Git, and our mailing lists. You may prefer to use an email account specifically for this purpose.</span>
 									</div>
 									<div class="hide spinner"></div>
+									
+									<div class="form-elements-container aggrement-container" id="aggrementContainer">
+										<input class="aggrement-field" type="checkbox" id="aggrement">I agree to the Eclipse.org <a href="https://www.eclipse.org/legal/termsofuse.php">Terms of Use</a> and <a href="https://www.eclipse.org/legal/privacy.php">Privacy Policy</a>.</>
+									</div>
```
Comment (carmacleod) [inline bundles/org.eclipse.orion.client.ui/web/mixloginstatic/register.html]: 4 x aggrement => agreement

File: bundles/org.eclipse.orion.client.ui/web/mixloginstatic/stylesheets/login.css [hunk 4]
```diff
@@ -478,6 +478,18 @@ input::-ms-reveal {
...
 
+.email-warning {
+	font-family: "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif;
+	font-size: 10px;
+	max-width: 330px;
+}
+
+.aggrement-container {
+    margin: 10px 0;
+    font-family: "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif;
```
Comment (carmacleod) [inline bundles/org.eclipse.orion.client.ui/web/mixloginstatic/stylesheets/login.css]: aggrement => agreement

File: bundles/org.eclipse.orion.client.ui/web/mixloginstatic/register.html (hunk 3 above)
Comment (carmacleod) [reply bundles/org.eclipse.orion.client.ui/web/mixloginstatic/register.html]: Wrap the input and the links in a label element, and delete the `</>`.
i.e.
```
<label><input class="aggrement-field" type="checkbox" id="aggrement">I agree to the Eclipse.org <a href="https://www.eclipse.org/legal/termsofuse.php">Terms of Use</a> and <a href="https://www.eclipse.org/legal/privacy.php">Privacy Policy</a>.</label>
```

**Pull request merged**